src/lib/utils/holidayUtils.test.ts

```diff
@@ -0,0 +1,28 @@
+import { getHolidayData, isTodayHoliday } from './holidayUtils';
+
+describe('isTodayHoliday', () => {
+	it('should return true if today is a holiday', () => {
+		const result = isTodayHoliday(new Date(2022, 11, 25));
+		expect(result).toBe(true);
+	});
+	it('should return false if today is not a holiday', () => {
+		const result = isTodayHoliday(new Date(2022, 0, 3));
+		expect(result).toBe(false);
+	});
+});
+
+describe('getHolidayData', () => {
+	it('should return holiday data if today is a holiday', () => {
+		const result = getHolidayData(new Date(2022, 11, 25));
+		expect(result).toEqual({
+			name: 'Christmas Day',
+			date: new Date(2022, 11, 25),
+			words: ['holly', 'yule', 'elf', 'frost', 'tree', 'eggnog', 'grinch'],
+			colors: ['from-red-400', 'to-green-500']
+		});
+	});
+	it('should return undefined if today is not a holiday', () => {
```

Comment:
microscopic nitpick: give these lads some breathing room
```suggestion
	});
	
	it('should return undefined if today is not a holiday', () => {
```